conjure-core/src/main/java/com/palantir/conjure/defs/validator/EnumValueDefinitionValidator.java

```diff
@@ -56,14 +57,14 @@ public void validate(EnumValueDefinition definition) {
 
     @com.google.errorprone.annotations.Immutable
     private static final class FormatValidator implements ConjureValidator<EnumValueDefinition> {
-        private static final Pattern REQUIRED_FORMAT = Pattern.compile("[A-Z][A-Z0-9]*(_[A-Z0-9]+)*");
+        private static final SimplifiedPattern REQUIRED_FORMAT = EnumPattern.get();
```

Comment:
imo this makes it easier to follow the code in an IDE:
```suggestion
        private static final EnumPattern REQUIRED_FORMAT = EnumPattern.get();
```